### ISSUE TITLE
Correctly follow links to doc sections

### ISSUE THREAD

**Issue by pfitzseb** [pfitzseb/DocSeeker.jl#3]

**Comment by zygmuntszpak** [comment]
When writing a docstring, If one writes something like

>  See also: [`Kernel.ando3`](@ref)   (i.e. `([Kernel.ando3](@ref)`  with backticks around Kernel.ando)


then currently the documentation browser will not find the function `ando3` defined in the `Kernel` module, but instead will match the first part of the word and try to bring up documentation for `Kernel` if it exists.  

Is this the bug that you are referencing here?

**Comment by pfitzseb** [comment]
No, this is about referencing section headers in e.g. Base's docs (or anything that's *not* a symbol, really). Do open another issue for the bug you have though :)
